cloud/performance.rst

```diff
@@ -86,3 +86,30 @@ To measure the number of active connections at a given time, run:
   SELECT COUNT(*)
     FROM pg_stat_activity
    WHERE state <> 'idle';
+
+PostgreSQL/PgBouncer Configuration
+==================================
+
+In addition to adding nodes and scaling their hardware, the Cloud Console allows changing database server configuration parameters. Adjusting the parameters can help tune the server for particular workloads. Cloud currently exposes just those parameters which do not require a server restart to take effect.
+
+To access these settings, go to the Configuration tab in your Cloud formation and click "Change a Parameter." It will present a dropdown list of config parameters:
+
+.. image:: ../images/cloud-pg-conf.png
+
+The options are grouped by which server and system they control:
+
+* Coordinator PostgreSQL
+* Coordinator Inbound PgBouncer
+* Worker PostgreSQL
+
+Selecting an option opens an input box that accepts the appropriate values, whether numerical, textual, or a pre-set list. The selected option also shows a link to learn more about the configuration parameter.
+
+.. image:: ../images/cloud-pg-conf-option.png
```

Comment:
It would be good if we can re-select the images. For example, we removed the `debug_print_plan` option because it can lead to oversized logs and out-of-disk issues.

Reply:
Done